src/libraries/Directory.Build.props

```diff
@@ -304,9 +308,13 @@
 
   <PropertyGroup Condition="'$(BuildAllConfigurations)' != 'true'">
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
-    <BinPlaceTestSharedFramework Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'">true</BinPlaceTestSharedFramework>
+    <BinPlaceTestSharedFramework Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS' or '$(TargetOS)' == 'Android'">false</BinPlaceTestSharedFramework>
```

Comment:
@dagood Is this too much to bite off?  Not creating testhost, for example, requires `build-job.yml` changes with respect to preparing testhost folder to publish.  I don't know other CI implications.

Reply:
I know less than you do here. 🙂 Not familiar with the libraries build, especially its tests.

Reply:
Ok, thanks.  I'm debating on punting even though it doesn't make sense to generate testhost for ios/android/tvos builds.